src/main/java/co/cask/cdc/plugins/source/sqlserver/CTInputDStream.java

```diff
@@ -180,24 +162,19 @@ private long getCurrentTrackingVersion(Connection connection) throws SQLExceptio
     return keyColumns;
   }
 
-  private List<TableInformation> getCTEnabledTables() {
-    Connection connection = dbConnection.apply();
+  private List<TableInformation> getCTEnabledTables(Connection connection) throws SQLException {
     List<TableInformation> tableInformations = new LinkedList<>();
     String stmt = "SELECT s.name as schema_name, t.name AS table_name, ctt.* FROM sys.change_tracking_tables ctt " +
       "INNER JOIN sys.tables t on t.object_id = ctt.object_id INNER JOIN sys.schemas s on s.schema_id = t.schema_id";
-    try {
-      ResultSet rs = connection.createStatement().executeQuery(stmt);
+    try(ResultSet rs = connection.createStatement().executeQuery(stmt)) {
```

Comment:
nit: add space between 'try' and '('

Reply:
I believe this is supposed to be caught by our checkstyle. It looks like this project doesn't have checkstyle setup though. Can you add it in a separate PR (See https://github.com/data-integrations/google-cloud for an example)